### DISCUSSION TITLE
Soll die Sie-Form oder die Du-Form verwendet werden?

### DISCUSSION THREAD

**Discussion by SpotlightForBugs** [Bootstrap-Academy/Bootstrap-Academy#39] [Q&A] [answered]
Momentan werden Nutzende der Deutschen Spracheinstellung gemixt gesiezt und geduzt. Eine Vereinheitlichung muss her, aber welche?

Ich finde lernen ist persönlich und deshalb ein "Du" wert, was denkt ihr ?

Dann muss natürlich auch die Meinung von TheMorpheus407 eingehalten werden, es geht um die Art und Weise, wie Nutzende behandelt werden sollen.

**Comment by SpotlightForBugs** [comment]
Du

**Comment by SpotlightForBugs** [comment]
Sie

**Comment by SpotlightForBugs** [comment]
Backlock/Frontlock: Den Nutzenden auswählen lassen (erhöhter Mehraufwand)

**Comment by goldjunge91** [comment]
Du-Form

**Comment by Defelo** [comment]
@TheMorpheus407 Denke mal, dass wir überall die Du-Form haben wollen, oder?

**Reply by TheMorpheus407** [reply, marked as answer]
Jup, Du-Form
